examples/wind/src/layers/particle-layer/particle-layer.js

```diff
@@ -332,24 +334,21 @@ export default class ParticleLayer extends Layer {
   }
 
   createTexture(gl, opt) {
```

Comment:
`createTexture(gl, opt = {}) {`
so we don't need the 2nd param in line 51 and 52